scrunch/datasets.py

```diff
@@ -173,6 +173,15 @@ def get_dataset(dataset, connection=None, editor=False, project=None):
         ds.change_editor(authenticated_url)
     return ds
 
+def get_project_by_id(project_id, connection=None):
+    """
+    :param project: Crunch project ID
+    :param connection: An scrunch session object
+    :return: Project class instance
+    """
+    connection = _default_connection(connection)
+    url = connection.session.site_url + "/projects/" +  project_id
```

Comment:
It's best to use urllib.urljoin for these, in this case there's little risk, but imagine someone sends `project_id = "../../datasets/something/"` which... sure, someone's just shooting themselves on the foot, but it's a bad practice in general to concatenate urls as strings.